### PR TITLE
Add ipytree

### PR DESCRIPTION
**Pull request by martinRenou**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ipytree/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
```
Comment (tadeu) [inline recipes/ipytree/meta.yaml]: `build:` should be changed to `host:` here, look here: https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

File: recipes/ipytree/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+    - pip
+    - jupyter
```
Comment (tadeu) [inline recipes/ipytree/meta.yaml]: Why is `jupyter` a host requirement (o.e., not only `ipywidgets`)

**Comment by martinRenou** [comment]
Thanks for the review @tadeu :)

**Comment by martinRenou** [comment]
@tadeu If everything's fine, I'd be grateful if you could merge it :)

**Comment by martinRenou** [comment]
Thanks @chrisburr :)

**Pull request merged**